hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/controller/ContractCallErrorResponse.java

```diff
@@ -0,0 +1,47 @@
+package com.hedera.mirror.web3.controller;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Data;
+
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonTypeName("_status")
+@Data
+public class ContractCallErrorResponse implements ContractCallResponse {
```

Comment:
This class is not specific to contract call and could be reused by all future REST APIs. Also, it doesn't make sense to have a `ContractCallResponse` interface and implement it for the same reason. This is just a generic error.

```suggestion
public class GenericErrorResponse {
```